package.json

```diff
@@ -6,6 +6,9 @@
   "repository": "git@github.com:lifeomic/chroma-react.git",
   "author": "LifeOmic <oss@lifeomic.com>",
   "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
```

Comment:
This is the solution semantic-release recommends for handling the `--access public` arg we were passing to `npm publish` before.

[docs](https://semantic-release.gitbook.io/semantic-release/support/faq#how-can-i-set-the-access-level-of-the-published-npm-package)